animal.go

```diff
@@ -88,17 +70,19 @@ func (g game) NextQuestion(q string, r string) (string, error) {
 }
 
 // Play - This is the actual game Play function
-func (g game) Play(r io.Reader, w io.Writer) error {
+func (g *game) Play(r io.Reader, w io.Writer) error {
 	question := StartingQuestion
 	var prev1, prev2 string
 
 	for g.Running {
 		fmt.Fprint(w, question, " ")
-		response, err := GetUserYesOrNo(r)
+		g.transcript.WriteString(question + " ")
```

Comment:
It's annoying to have to keep repeating every string, once to the writer and once to the transcript, isn't it? And, as we've seen, it's easy to write bugs where you write something to the writer but not the transcript, or vice versa. Maybe we should add some Game method which just writes to both automatically?

Reply:
Great idea, implemented this change.